scripts/seed-db.sh

```diff
@@ -0,0 +1,14 @@
+set -e
+
+set -o allexport
+source .env
+set +o allexport
+
+echo "Using .env.LOCAL_WEB_SERVER=$LOCAL_WEB_SERVER"
+
+echo "\nClearing current db rows"
+curl -s "$LOCAL_WEB_SERVER/api/admin/rmrf" > /dev/null
+echo "\nSeeding with test data"
+curl "$LOCAL_WEB_SERVER/api/admin/seed"
```

Comment:
I'm getting the below error at this step. Is there something I need to reset in my hasura environment after starting it with the CLI?

```
{
  errors: [
    {
      extensions: [Object],
      message: 'invalid x-hasura-admin-secret/x-hasura-access-key'
    }
  ]
}
API ERROR Error:
    at new GraphQLError (/home/kevin/Research/coordinape/coordinape-frontend/src/lib/gql/zeusHasuraAdmin/index.ts:15797:5)
    at /home/kevin/Research/coordinape/coordinape-frontend/src/lib/gql/zeusHasuraAdmin/index.ts:16268:17
    at processTicksAndRejections (internal/process/task_queues.js:95:5) {
  response: { errors: [ [Object] ] }
}
```

Reply:
I'm stuck at this point as well

Reply:
turns out I had HASURA_ADMIN_SECRET in my .env file twice. The proper entry is:
`HASURA_ADMIN_SECRET=admin-secret`